lib/helper.js

```diff
@@ -225,6 +226,86 @@ class Helper {
   static isNumber(obj) {
     return typeof obj === 'number' || obj instanceof Number;
   }
+
+  /**
+   * @static
```

Comment:
we don't use `@static`, `@throws` and `@example` declarations. They are harmless, but I'd prefer them to be removed for consistency with the rest of the code.